### DISCUSSION TITLE
ACF Repeater collapse field not shown

### DISCUSSION THREAD

**Discussion by karlbaumhauer** [flyntwp/flynt#368] [Q&A] [answered]
hi,

I opened this support question in the[ ACF forum](https://support.advancedcustomfields.com/forums/topic/repeater-collapse-field-not-shown/) and they sent me to you... 

here I just copy-paste my question. Pls read the thread in the link to see answers from the ACF user...

I am using ACF pro and WordPress 5.7 with the flynt base theme and try to display the second sub_field if collapsed:

[ 
    'label' => __('Items', 'flynt'),
    'name' => 'items',
    'type' => 'repeater',
    'layout' =>
    'block',
    'collapse' => 'headline',
    'min' => 1,
    'max' => 4,
    'button_label' => 'Hinzufügen',
    'sub_fields' => [
        [ 
            'label' => __('Icon', 'flynt'),
            'name' => 'icon',
            'type' => 'text',
        ],
        [ 
            'label' => __('Headline', 'flynt'),
            'name' => 'headline',
            'type' => 'text',
        ],
        [ 
            'label' => __('Text', 'flynt'),
            'name' => 'text',
            'type' => 'wysiwyg',
            'toolbar' => 'basic',
            'media_upload' => 0, 'delay' => 1,
        ],
    ],
],

Any ideas why it never shows anything?

So far values I tried: 0, 1, ”, ‘icon’, ‘text’ & ‘headline’

**Comment by thomasnavarro** [comment, marked as answer]
Hi,
I had the same problem, while searching a little I found something that could help you. The key to use in the "collapsed" field of the repeater is generated at [this line in the acf-field-group-composer plugin](https://github.com/flyntwp/acf-field-group-composer/blob/master/lib/ACFComposer/ResolveConfig.php#L119) and should look like this: `field_pageComponents_HeroSlider_slides_boTitle`.

I hope this can help you ;)

**Reply by thomasnavarro** [reply]
![Exemple](https://user-images.githubusercontent.com/36193187/115379641-03917300-a1d2-11eb-9e35-8f991d98d0cc.jpg)

**Reply by karlbaumhauer** [reply]
@ThomasNavarro you are awesome...!!! this totally works and after so many days of research and testing I finally got it. Thx a lot!!!

@steffenbew, @domtra this is a nice solution if somebody else will ask.
